src/main/resources/Documentation/about.md

```diff
@@ -98,12 +98,71 @@ or by editing project configuration using web ui.
 
 Plugin configuration as part of the global configuration can be edited through the gerrit.config file.
 
-Message content and commit message parsing
+Message content
 ------------------------------------------
 
-TODO!
+When a patchset is pushed, the plugin will parse commit info and form a EiffelSourceChangeCreatedEvent that will contain the following fields.
+
+* repository name
+* branch name
+* repository url
+* author name
+* author username
+* author e-mail
+* commit id
+* change id
+* line insertions
+* lines deletions
+
+There will also be a EiffelSourceChangeSubmittedEvent when a patchset is submitted containing the following fields.
+
+* repository name
+* branch name
+* repository url
+* author name
+* author username
+* author e-mail
+* commit id
 
 Eiffel Event Linking
 --------------------
 
-TODO: Explain event linking and show examples.
+When a patchset is pushed to a gerrit repository and branch the EiffelSourceChangeCreatedEvent that is created will have the
+BASE link set to the previous EiffelSourceChangeSubmittedEvent. If there is no previous EiffelSourceChangeSubmittedEvent 
+for that particular repository and branch then no BASE link will be set. Any following patchsets pushed to the same review will
+be connected in a one way chain from newest to oldest through the PREVIOUS_VERSION link.
+When it's time to submit the patchset an EiffelSourceChangeSubmittedEvent is created with a CHANGE link set to the last
+patchset pushed to that particular review and a PREVIOUS_VERSION link set to the last EiffelSourceChangeSubmittedEvent
+for that repository and branch. If there is no previous EiffelSourceChangeSubmittedEvent then no PREVIOUS_VERSION link will be set.
```

Comment:
Is it easier to link to the scenario documentation and maybe move it?

Reply:
Sure, I will link to the scenarios doc.